packages/jest-expect/src/index.ts

```diff
@@ -6,27 +6,30 @@
  */
 
 import type {Config} from '@jest/types';
-import {Expect, expect} from 'expect';
+import {expect} from 'expect';
 import {
   addSerializer,
   toMatchInlineSnapshot,
   toMatchSnapshot,
   toThrowErrorMatchingInlineSnapshot,
   toThrowErrorMatchingSnapshot,
 } from 'jest-snapshot';
+import type {JestExpect} from './types';
 
-export default function jestExpect(
-  config: Pick<Config.GlobalConfig, 'expand'>,
-): Expect {
-  expect.setState({expand: config.expand});
-  expect.extend({
+export type {JestExpect} from './types';
+
+export function createJestExpect({expand}: Config.GlobalConfig): JestExpect {
+  const jestExpect = expect as JestExpect;
```

Comment:
should we somehow make a copy instead of mutating? doesn't really matter of course

Reply:
Feels like this got solved too.